lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/client/REEFImplementation.java

```diff
@@ -96,11 +96,11 @@ public void submit(final Configuration driverConf) {
       final JobSubmissionEvent submissionMessage;
       try {
         if (this.clientWireUp.isClientPresent()) {
-          submissionMessage = this.jobSubmissionHelper.getJobSubmissionBuilder(driverConf)
+          submissionMessage = this.jobSubmissionHelper.getJobSubmissionBuilder(driverConfiguration)
```

Comment:
This fixes a genuine bug in `REEFImplementation`, right? We should call this out in the commit message.